docs/source/config/integrating.rst

```diff
@@ -44,9 +44,10 @@ Custom exception tracebacks
 ===========================
 
 Rarely, you might want to display a different traceback with an exception -
```

Comment:
Rarely, you might want to display a custom traceback when reporting an exception. To do this, define the custom traceback using ``_render_traceback_(self)`` method which returns a list of strings, one string for each line of the traceback. For example, the `ipyparallel <http://ipyparallel.readthedocs.io/>`__ computing framework uses this to display errors from multiple engines.